include/seqan3/alphabet/cigar/exposition_only/cigar_operation.hpp

```diff
@@ -0,0 +1,104 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \brief Introduces the seqan3::exposition_only::cigar_operation alphabet.
+ * \author Joshua Kim <joshua.kim AT fu-berlin.de>
+ */
+
+#pragma once
+
+#include <seqan3/alphabet/alphabet_base.hpp>
+
+// ------------------------------------------------------------------
+// cigar_operation
+// ------------------------------------------------------------------
+
+namespace seqan3::exposition_only
+{
+
+/*!\brief The actual implementation of seqan3::cigar::operation for documentation purpose-only.
+ * \implements seqan3::writable_alphabet
+ * \if DEV \implements seqan3::detail::writable_constexpr_alphabet \endif
+ * \implements seqan3::trivially_copyable
+ * \implements seqan3::standard_layout
+ * \implements std::regular
+ *
+ * \details
+ *
+ * \note Please note that this class only exists, because of technically reasons.
+ *       Please always use seqan3::cigar::operation instead of seqan3::exposition_only::cigar_operation.
+ *
+ * \if DEV
+ * \note We can't declare seqan3::cigar::operation in-class, because we need to specify the second tuple element of
+ *       seqan3::alphabet_tuple_base before we actually can declare it in-class. This is a trade-off to make
+ *       seqan3::cigar a non-template class.
```

Comment:
```suggestion
 * \note We cannot declare seqan3::cigar::operation in-class, because we need to specify the second tuple element of
 *       seqan3::alphabet_tuple_base before we actually can declare it in-class. This is a trade-off to make
 *       seqan3::cigar a non-template class.
```